Gemfile

```diff
@@ -46,7 +46,7 @@ gem "ims-lti", "~> 2.2.3"
 gem "local_time", "~> 2.0"
 
 gem "oauth",           "~> 0.5.4"
-gem "octicons_helper", "~> 2.1"
+gem "octicons_helper", "~> 9.1.1"
```

Comment:
Updated because our old set didn't have the kebab-horizontal that was needed for the menu design.